aries_cloudagent/protocols/present_proof/v2_0/routes.py

```diff
@@ -127,6 +127,11 @@ class V20PresProposalByFormatSchema(OpenAPISchema):
         required=False,
         metadata={"description": "Presentation proposal for DIF"},
     )
+    vc_di = fields.Nested(
+        DIFProofProposalSchema,
```

Comment:
shouldn't this be VCDI?

Reply:
I didn't want to write another proposal schema since DIFProofProposalSchema already exist to avoid redundancy

Reply:
Also reference to  https://github.com/hyperledger/aries-rfcs/blob/main/features/0510-dif-pres-exch-attach/README.md